lib/Doctrine/ORM/Query/Parser.php

```diff
@@ -470,6 +470,11 @@ public function semanticalError($message = '', $token = null)
     {
         if ($token === null) {
             $token = $this->lexer->lookahead;
```

Comment:
This one can be simplified with the `??` operator, like:

```php
$token = $this->lexer->lookahead ?? ['position' = null];
```

Reply:
Done.